_tasks/test_browser/helpers.ts

```diff
@@ -0,0 +1,17 @@
+export const pathToJS = (path: string) => path.replace(".ts", ".js")
```

Comment:
Always treat common abbreviations as words. Aka., the casing of `JS` in the context of `pathToJS` should be `pathToJs`. Same thing with `pathToHTML`.